src/back/controllers/api/routes/challenges/post.js

```diff
@@ -0,0 +1,17 @@
+'use strict';
+
+const Challenge = require('../../models/challenge');
+
+const post = (req, res) => {
+    const challenge = new Challenge(req.body);
```

Comment:
mixing business logic with transfer protocol violates single responsibility principle
please keep route handler as slim as possible, it's just connecting layer between transfer protocol and business logic.